package.json

```diff
@@ -13,7 +13,7 @@
   },
   "author": "ACME Inc.",
   "scripts": {
-    "test": "./test/spec.rb",
+    "test": "mocha test/*test.js",
```

Comment:
Coming from a sort of BDD school of thoughts I'm pretty attached to `*.spec.js`. That seems to be pretty common in the Mocha community too.